src/NServiceBus.RavenDB.AcceptanceTests/Config.cs

```diff
@@ -12,8 +12,7 @@ public void Configure(BusConfiguration config)
         documentStore = new DocumentStore
         {
             Url = "http://localhost:8083",
-            DefaultDatabase = Guid.NewGuid().ToString(),
-            ResourceManagerId = Guid.NewGuid() /* This is OK for ATT purposes */
```

Comment:
Oh, no, I **did** mean to do this. ResourceManagerId should be taken care of properly for all docstores with this PR, so I don't *THINK* it's necessary to specify it. Or if it was, it would get overridden by the new code anyway.